packages/webapp/pages/_app.tsx

```diff
@@ -34,14 +34,14 @@ import { PushNotificationContextProvider } from '@dailydotdev/shared/src/context
 import { useThemedAsset } from '@dailydotdev/shared/src/hooks/utils';
 import { DndContextProvider } from '@dailydotdev/shared/src/contexts/DndContext';
 import { structuredCloneJsonPolyfill } from '@dailydotdev/shared/src/lib/structuredClone';
-import { fromCDN } from '@dailydotdev/shared/src/lib';
 import { useOnboarding } from '@dailydotdev/shared/src/hooks/auth';
 import { useCheckCoresRole } from '@dailydotdev/shared/src/hooks/useCheckCoresRole';
 import {
   messageHandlerExists,
   postWebKitMessage,
   WebKitMessageHandlers,
 } from '@dailydotdev/shared/src/lib/ios';
+import { fromCDN } from '@dailydotdev/shared/src/lib';
```

Comment:
Same here, move it back to its original position

Reply:
Done ✔️

Reply:
nit-picking, nit-picking everywhere